strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/testing/TestCaseWithMavenArtifactGenerationAndIndexing.java

```diff
@@ -141,43 +141,43 @@ protected void createRepository(String storageId,
                                     boolean indexing)
             throws IOException, JAXBException, RepositoryManagementStrategyException
     {
-        MutableMavenRepositoryConfiguration repositoryConfiguration = new MutableMavenRepositoryConfiguration();
+        MavenRepositoryConfigurationDto repositoryConfiguration = new MavenRepositoryConfigurationDto();
         repositoryConfiguration.setIndexingEnabled(indexing);
 
         createRepository(storageId, repositoryId, RepositoryPolicyEnum.RELEASE.getPolicy(), repositoryConfiguration);
     }
 
     protected void createRepository(String storageId,
                                     String repositoryId,
-                                    MutableMavenRepositoryConfiguration repositoryConfiguration)
+                                    MavenRepositoryConfigurationDto repositoryConfiguration)
             throws IOException, JAXBException, RepositoryManagementStrategyException
     {
         createRepository(storageId, repositoryId, RepositoryPolicyEnum.RELEASE.getPolicy(), repositoryConfiguration);
     }
 
-    protected MutableRepository createRepository(String storageId,
+    protected RepositoryDto createRepository(String storageId,
                                                  String repositoryId,
                                                  String policy,
                                                  boolean indexing)
         throws IOException,
         JAXBException,
         RepositoryManagementStrategyException
     {
-        MutableMavenRepositoryConfiguration repositoryConfiguration = new MutableMavenRepositoryConfiguration();
+        MavenRepositoryConfigurationDto repositoryConfiguration = new MavenRepositoryConfigurationDto();
         repositoryConfiguration.setIndexingEnabled(indexing);
 
         return createRepository(storageId, repositoryId, policy, repositoryConfiguration);
     }
 
-    protected MutableRepository createRepository(String storageId,
+    protected RepositoryDto createRepository(String storageId,
                                                  String repositoryId,
                                                  String policy,
-                                                 MutableMavenRepositoryConfiguration repositoryConfiguration)
+                                                 MavenRepositoryConfigurationDto repositoryConfiguration)
```

Comment:
Cosmetic: Could you please re-indent the method parameters? Thanks! :)

Reply:
fixed